CONTRIBUTING.md

```diff
@@ -0,0 +1,105 @@
+# Contributing to The Show Case App
+
+We would love for you to contribute to the project and help make it even better! You can contribute either through code or helping us out on different issues, suggesting features etc. As a contributor here is the guidelines we would like you to follow:
+
+
+ - [Code of Conduct](#coc)
+ - [Question or Problem?](#question)
+ - [Issues and Bugs](#issue)
+ - [Feature Requests](#feature)
+ - [Submission Guidelines](#submit)
+
+## <a name="coc"></a> Code of Conduct
+Help us keep the project open and inclusive. Please read and follow our (Code of Conduct)[https://github.com/UdacityMobileWebScholarship/showcase-app/blob/master/CODE_OF_CONDUCT.md].
+
+## <a name="question"></a> Got a Question, Query or Problem?
+
+Do not open issues for general queries as we want to keep GitHub issues for bug reports, feature requests and project development related contributions. For general queries reach out to us on [#cp_shwcase_app](https://mobilewebindiascholar.slack.com/messages/CA8MRBPAT) channel on slack.
+
+## <a name="issue"></a> Found a Bug?
+If you find a bug in the source code, you can help us by
+[submitting an issue](https://github.com/UdacityMobileWebScholarship/showcase-app/issues/new) to our GitHub Repository. Even better, you can check out issues [open for claim](https://github.com/UdacityMobileWebScholarship/showcase-app/labels/open%20for%20claim) and claim one!
+
+## <a name="feature"></a> Missing a Feature?
+You can *request* a new feature by [submitting an issue](https://github.com/UdacityMobileWebScholarship/showcase-app/issues/new) to our GitHub
+Repository. Given the project timeline and necessity for the feature, we'll discuss the roadmap accordingly. If you would like to *implement* a new feature, please submit an issue with
+a proposal for your work first, to be sure that we can use it.
+
+## <a name="submit"></a> Submission Guidelines
+
+### <a name="submit-issue"></a> Submitting an Issue
+Before you submit an issue, please search the issue tracker, maybe an issue for your identified problem already exists and you can join the discussion there only.
+For bugs, please try to provide as much information possible to reproduce it as possible, that will really help us a lot!
+
+### <a name="submit-pr"></a> Submitting a Pull Request (PR)
+1. Make sure there is an issue open for the PR and it is assigned to you before starting any work. That way it'll be helpful to eliminate duplicate efforts.
+2. Fork the repo.
+3. Make your changes in a new git branch:
+
+     ```shell
+     git checkout -b my-fix-branch master
```

Comment:
We should add a note here, asking the collaborator to update his master once again before checking out the feature branch. Many times people forget to do this. I am also a culprit sometimes. :)

Reply:
Good point. Totally agree!